README.md

```diff
@@ -38,6 +38,8 @@ Prerequisites:
 
 - Open [src\Calculator.sln](/src/Calculator.sln) in Visual Studio to build and run the Calculator app.
 - For a general description of the Calculator project architecture see [ApplicationArchitecture.md](docs/ApplicationArchitecture.md).
+- To run the UI Tests, you need to install
+  [Windows Application Driver (WinAppDriver)](https://github.com/microsoft/WinAppDriver#installing-and-running-windows-application-driver).
```

Comment:
The instructions on this page are misleading (we don't need to run WinAppDriver.exe manually).

Here is a direct link to the last stable release: https://github.com/microsoft/WinAppDriver/releases/latest

Reply:
Fair assessment.  Will switch to your releases link instead.  Thanks @rudyhuyn.